data-structures.md

```diff
@@ -70,7 +91,20 @@ type Block struct {
 }
 ```
 
-#### Sharded Messages and Receipts
+### Element Ordering in Arrays
+
+As you see in the interface defined above, a typical block header contains many different arrays. The ordering of these arrays is important and should be consistent across implementations.
+
+| Array | Ordering |
+| --- | --- |
+| `Tickets` | Sorted by the order in which the tickets were created (each ticket in the array is created at a specific block height). |
+| `Parents` | Sorted lexicographically |
```

Comment:
This contradicts https://github.com/filecoin-project/specs/issues/285#issuecomment-497795865 and some places in the spec where the ordering is explicitly undefined (#289)